XmacsLabs/mogan
# remove improper prog mode in error output and prompt

## What

Replace improper prog mode in error output and prompt with font definitions in scheme session

在scheme会话中，从错误输出和命令提示符处，将不正确的prog模式定义替换为等价的字体定义

## Why

Prompt is characters in front of instructions, such as `Scheme]` in scheme and `(%o1)` in maxima. Prompt may or may not be valid program expressions in corresponding language.

## How to test your changes?

1. clean style package cache in `get_tm_cache_path()`
2. open research, insert scheme session, then see if color at prompt is rendered properly (green)
3. raise error from session by `(car 1)` then see if color of error is rendered properly (red)
